hexa/workspaces/tests/test_schema/test_workspace.py

```diff
@@ -97,6 +97,13 @@ def setUpTestData(cls):
             role=WorkspaceMembershipRole.VIEWER,
             status=WorkspaceInvitationStatus.ACCEPTED,
         )
+        cls.INVITATION_BAR = WorkspaceInvitation.objects.create(
```

Comment:
Shouldn't it use another name than the one just above ?

Reply:
Yes it should be BAZ. My bad